Pokedex/Main/MainViewModel.swift

```diff
@@ -0,0 +1,48 @@
+//
+//  MainViewModel.swift
+//  Pokedex
+//
+//  Created by krikaz on 6/25/20.
+//  Copyright © 2020 thewire. All rights reserved.
+//
+
+import Foundation
+
+final class MainViewModel {
+    private let client = PokemonClient()
+    private var pokemons: [Pokemon]? {
+        didSet {
+            self.pokemonsOutput?((self.pokemons?.map { $0.name })!)
+        }
+    }
+    var nextPokemons: String?
```

Comment:
Try to use explicit name for variables so you can know what it's supposed to be just by the name. What kind of String is `nextPokemons` supposed to represent ? An URL ? `nextPokemonsURLString` would be more explicit for example.